### ISSUE TITLE
[Adjust Rule] SIM401 unnecessarily quotes default value

### ISSUE THREAD

**Issue by akx** [MartinThoma/flake8-simplify#89]
## Desired change

* **Rule(s)**: SIM401
* **Adjustment**: The default value should not be quoted.
* **Version**: flake8-simplify==0.14.5

## Explanation

The pattern

```
if a:
    token = a[1]
elif 'token' in dct:
    token = dct['token']
else:
    token = None
```

is reported as

```
SIM401 Use 'token = dct.get(token, "None")' instead of an if-block
```

though it obviously should be
```
SIM401 Use 'token = dct.get(token, None)' instead of an if-block
```
or when the default is None,
```
SIM401 Use 'token = dct.get(token)' instead of an if-block
```

## Example

See above.

**Comment by MartinThoma** [comment]
hi @akx , Thank you for reporting this issue. I'm currently fixing several issues with SIM401 in PR #85 . This

**Comment by MartinThoma** [comment]
Closed by https://github.com/MartinThoma/flake8-simplify/pull/85

**Comment by MartinThoma** [comment]
Thank you very much for your help. The issue was fixed in `flake8-simplify==0.14.6`
